selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/miscellaneous/CheckConvertingToMavenProjectTest.java

```diff
@@ -0,0 +1,130 @@
+import org.eclipse.che.selenium.core.client.TestProjectServiceClient;
+import org.eclipse.che.selenium.core.constant.TestMenuCommandsConstants;
+import org.eclipse.che.selenium.core.workspace.TestWorkspace;
+import org.eclipse.che.selenium.pageobject.AskForValueDialog;
+import org.eclipse.che.selenium.pageobject.CodenvyEditor;
+import org.eclipse.che.selenium.pageobject.Ide;
+import org.eclipse.che.selenium.pageobject.InformationDialog;
+import org.eclipse.che.selenium.pageobject.Loader;
+import org.eclipse.che.selenium.pageobject.Menu;
+import org.eclipse.che.selenium.pageobject.ProjectExplorer;
+import org.eclipse.che.selenium.pageobject.Wizard;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+/** @author Aleksandr Shmaraev */
+public class CheckConvertingToMavenProjectTest {
+  private static final String PROJECT_NAME = NameGenerator.generate("project", 4);
+  private static final String WEB_APP_MODULE = "my-webapp";
+  private static final String NONE_MAVEN_PROJECT = NameGenerator.generate("noneMavenProject", 4);
+  private static final String PARENT_INFORMATION =
+      "<parent>\n"
+          + "<groupId>org.eclipse.che.examples</groupId>\n"
+          + "<artifactId>qa-multimodule</artifactId>\n"
+          + "    <version>1.0-SNAPSHOT</version>\n"
+          + "</parent>\n";
+  @Inject private TestWorkspace workspace;
+  @Inject private Ide ide;
+  @Inject private ProjectExplorer projectExplorer;
+  @Inject private CodenvyEditor editor;
+  @Inject private Menu menu;
+  @Inject private Loader loader;
+  @Inject private Wizard wizard;
+  @Inject private AskForValueDialog askForValueDialog;
+  @Inject private InformationDialog informationDialog;
+  @Inject private ActionsFactory actionsFactory;
+  @Inject private TestProjectServiceClient testProjectServiceClient;
+  @Inject private SeleniumWebDriver seleniumWebDriver;
+  private String workspaceId;
+
+  @BeforeClass
+  public void setUp() throws Exception {
+    workspaceId = workspace.getId();
+    URL mavenMultimodule = getClass().getResource("/projects/java-multimodule");
+    URL noneMavenProject = getClass().getResource("/projects/console-cpp-simple");
+    testProjectServiceClient.importProject(
+        workspaceId, Paths.get(mavenMultimodule.toURI()), PROJECT_NAME, UNDEFINED_PROJECT);
+    testProjectServiceClient.importProject(
+        workspaceId, Paths.get(noneMavenProject.toURI()), NONE_MAVEN_PROJECT, UNDEFINED_PROJECT);
+
+    ide.open(workspace);
+    ide.waitOpenedWorkspaceIsReadyToUse();
+    projectExplorer.quickRevealToItemWithJavaScript(format("%s/%s", PROJECT_NAME, WEB_APP_MODULE));
+  }
+
+  @Test
+  public void shouldConvertToMavenMultimoduleProject() throws Exception {
+    convertPredefinedFolderToMavenProjectWithContextMenu(
+        format("%s/%s", PROJECT_NAME, WEB_APP_MODULE));
+    testProjectServiceClient.checkProjectType(
+        workspaceId, format("%s/%s", PROJECT_NAME, WEB_APP_MODULE), "maven");
```

Comment:
What about move all these items to the upper of the test and declare them as constants.
```
final String convertPath = format("%s/%s", PROJECT_NAME, WEB_APP_MODULE);
```